scripts/zones/Beadeaux_[S]/mobs/DaDha_Hundredmask.lua

```diff
@@ -1,25 +1,16 @@
 -----------------------------------
 -- Area: Beadeaux [S] (92)
---  NM:  Da'Dha Hundredmask
--- !spawnmob 17154195
+--   NM: Da'Dha Hundredmask
 -- !pos -89.901 .225 -159.694 92
 -----------------------------------
+mixins = {require("scripts/mixins/job_special")}
+require("scripts/globals/status")
+-----------------------------------
 
 function onMobSpawn(mob)
-    mob:setLocalVar("HpTrigger", math.random(5,65));
-    mob:setMod(dsp.mod.TRIPLE_ATTACK, 25);
-    mob:addMod(dsp.mod.GRAVITYRES, 50);
-end;
-
-function onMobFight( mob, target )
-    if (mob:getHPP() < mob:getLocalVar("HpTrigger") and mob:getLocalVar("perfect_dodge") == 0) then
-        mob:useMobAbility(1013);
-        mob:setLocalVar("perfect_dodge", 1);
-    end
-end;
+    mob:setMod(dsp.mod.TRIPLE_ATTACK, 25)
```

Comment:
merging coz it was already present but now that I see it highlighted I wonder if mods here are legit retail or someoen didn't realize job traits apply to mobs /shrug